src/resources/users/user-router.js

```diff
@@ -1,22 +1,28 @@
+require('dotenv');
 const router = require('express').Router();
 
 const userController = require('./user-controllers');
 const checkAuth = require('../../utils/check-auth');
 const userMiddleware = require('../users/user-helpers');
-
 const {
   validateId,
   validateRegister,
   validateLogin,
   validatePasswordUpdate,
 } = userMiddleware;
 
+router.get('/accountRecovery', userController.accountRecovery);
 router.get('/', checkAuth, userController.getUsers);
 router.get('/:id', checkAuth, userController.getUserByID);
 router.post('/register', validateRegister, userController.register);
 router.post('/login', validateLogin, userController.login);
 router.delete('/:id', checkAuth, validateId, userController.delete);
-router.put('/settings', validatePasswordUpdate, userController.putSettings);
```

Comment:
You don't can change routes because we are using them already on the frontend. You need to create a new route if you dont can use this one